src/nevermined/Assets.ts

```diff
@@ -136,6 +198,7 @@ export class Assets extends Instantiable {
         erc20TokenAddress?: string,
         providers: string[] = [this.config.neverminedNodeAddress],
         appId?: string,
+        publishMetadata: PublishMetadata = PublishMetadata.JustMarketplaceAPI,
```

Comment:
I think this should actually be `PublishMetadata[] = []`. marketplace api is always use and we allow to upload the same ddo to multiple immutable backends

Reply:
Not sure about that one. Probably I don't see a real use case where you need to use IPFS & Filecoin at the same time for example. Not sure Marketplace API needs to be there, there could be use cases where is not needed a discovery function. 
Having also more than one immutable url would require modify the contracts. 
I suggest to discuss that out of the current issue.